conn_unix.go

```diff
@@ -396,13 +393,15 @@ func (c *Conn) writev(in [][]byte) (int, error) {
 	}
 
 	if len(in) > 1 && size <= 65536 {
-		b := mempool.Malloc(size)
+		b := c.g.writeBufferAllocator.Malloc(size)
 		copied := 0
 		for _, v := range in {
 			copy(b[copied:], v)
 			copied += len(v)
 		}
-		return c.write(b)
+		n, err := c.write(b)
+		c.g.writeBufferAllocator.Free(b)
```

Comment:
Is this a leak fix?

Reply:
No, it was not a leak, we free buffer after conn.write in previous version:
https://github.com/lesismal/nbio/blob/master/conn_unix.go#L76

But in this version, who call conn.write should free the buffer itself.